samples/README.md

```diff
@@ -0,0 +1,125 @@
+# Samples
+
+The samples are intended to demonstrate how different scenarios are covered by
+the OpenTelemetry .NET Auto-Instrumentation.
+
+## Overview
+
+The sample script will instrument two .NET processes, these process are identified
+as two [OpenTelemetry services](https://github.com/open-telemetry/opentelemetry-specification/blob/d6bcc0cb072d8d6f6ced856f1f23c451648a3caa/specification/resource/semantic_conventions/README.md#service)
+with names:
+
+ 1. **`aspnet-server`**: an ASP.NET Core application that makes requests to MongoDB and Redis databases;
+ 2. **`http-client`**: a .NET application that makes a few HTTP requests to the `aspnet-server` and some public websites;
+
+The script creates docker containers for MongoDB, Redis, and Jaeger.
+You can see the traces generated by the sample at the Jaeger container.
+The number and organization of traces generated depends on the `http-client`
+used in the specific run.
+The script will wait for your input before stopping the docker containers.
+
+## How to Run the Samples
+
+ 1. Create a local build of the project per [developing instructions](../docs/developing.md)
+ 2. On the root of the repository run the `./run_sample.sh`
+
+The script run can be customized via environment variables:
+
+| Environment variable | Description | Default |
+|-|-|-|
+| **aspNetAppTargetFramework** | Target framework for the `aspnet-server` service | `netcoreapp3.1` |
+| **exporter** | Value for [`OTEL_TRACES_EXPORTER`](../docs/config.md#exporters) env. variable | `jaeger` |
+| **sampleApp** | Application selected for the `http-client` service | `ConsoleApp` |
+| **sampleAppInjectSDK** | Boolean to control if Auto-instrumentation will inject the SDK on the `http-client` service | `true` |
+| **sampleAppTargetFramework** | Target framework for the `http-client` service | `netcoreapp3.1` |
+
+Usage example:
+```terminal
+$ sampleAppTargetFramework="net6.0" ./run-sample.sh
+```
+
+## Sample Projects
+
+Description of each project under the `samples` folder.
```

Comment:
```suggestion
You can find a description of each project in the `samples` folder.
```

Reply:
This gets a bit recursive since the file with the description is this one. 🤔 

```suggestion
You can find a description of each sample project below.
```